.github/workflows/update-bidi-types.yml

```diff
@@ -0,0 +1,67 @@
+# This workflow will update the WebdriverBidi types.
+
+name: Update WebdriverBidi types
+
+on:
+  schedule:
+    # Run hourly at xx:30.
+    - cron: '30 * * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build WebDriverBidi types
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+        with:
+          repository: w3c/webdriver-bidi
+      - name: Set up Rust
+        run: rustup update stable && rustup default stable
+      - name: Install cddlconv
+        run: cargo install --git https://github.com/jrandolf/cddlconv.git cddlconv
+      - name: Generate CDDL types
+        run: ./scripts/test.sh
+      - name: Generate TypeScript types
+        run: cddlconv all.cddl > webdriver-bidi.ts
```

Comment:
It would be nice to have the ability to generate types locally (for debugging, and as an escape hatch).

Consider moving the `cddlconv` and `test.sh` commands to a script in `package.json`.

Reply:
This is related to the above. See the comment.

Reply:
I think it's independent-ish (it creates more work as in one needs to manually clone the upstream repository, but having the ability to regenerate the types is still useful).